index.html

```diff
@@ -2039,36 +2039,87 @@ <h3>Media Types</h3>
+               the representations of both a <a>Consumer</a> (<a>Consumed Thing</a>) 
+               and a <a>Thing</a> (<a>Exposed Thing</a>).</p>
+
+                <figure id="servient-intermediary">
+                    <img
+                        src="images/servient-intermediary.png" />
+                    <figcaption>Servient as an Intermediary</figcaption>
+                </figure>
+
+            <p>This section illustrates using system configuration diagrams 
+               how <a>Servients</a> being WoT system components with software objects 
+               <a>Exposed Things</a> and <a>Consumed Things</a> work together 
+               make systems based on WoT architecture.</p>
 
             <section>
                 <h3>Direct Communication</h3>
                 <p>
                     <a href="#high-level-application-device"></a> shows
-                    direct communication between a <a>WoT Server</a>, that is
-                    exposing a thing and a <a>WoT Client</a> that is using it.
-                    Direct communication applies when both components
+                    direct communication between a <a>Thing</a>, that is
+                    exposing <a>interaction affordances</a> and a <a>Consumer</a> that is using it.
+                    Direct communication applies when both WoT components
                     use the same network protocol(s) and are accessible
                     to each other.
                 </p>
                 <figure id="high-level-application-device">
                     <img
                         src="images/arch-high-level-application-device-alt2.png" />
                     <figcaption>High level architecture of
-                        application and device</figcaption>
+                        Consumer and Thing</figcaption>
                 </figure>
 
-                Conceptually, a <a>WoT Server</a> can be thought to have an <a>Exposed Thing</a>
-                that provides an interface that conforms to the TD. A
-                <a>WoT Client</a> can be considered to contain a <a>Consumed Thing</a>
-                for each of the <a>WoT Servers</a> it connects to
+                Conceptually, a <a>Thing</a> can be thought to have an <a>Exposed Thing</a>
+                that provides an interface that conforms to the TD. 
+                A <a>Consumer</a> can be considered to contain a <a>Consumed Thing</a>
+                for each of the <a>Things</a> it connects to
                 that provides a programming interface that can be used
-                by applications. A <a>WoT Client</a> can generate a
+                by applications. A <a>Consumer</a> can generate a
                 <a>Consumed Thing</a> instance by interpreting a TD.
-                Communication between the <a>WoT Client</a> and the <a>WoT Server</a>
+                Communication between the <a>Consumer</a> and the <a>Thing</a>
```

Comment:
If we stick to the separation of abstraction and implementation, there is **interaction** between Consumer and Thing, and **communication** between Servients or Consumed and Exposed Thing.